bin/git-forgit

```diff
@@ -470,6 +470,20 @@ _forgit_stash_push() {
     _forgit_git_stash_push ${msg:+-m "$msg"} -u "${files[@]}"
 }
 
+_forgit_clean_preview() {
+    local path
+    path=$1
+    if [[ -d "$path" ]]; then
+        if hash tree &> /dev/null; then
+            tree "$path"
```

Comment:
Should this binary be set as a variable one can control? similar to PAGER or DELTA etc?

Are there other places in the code we care about looking at directories with a tool like this? (if the answer is either or both, we could consider accepting this change as is, then making it configurable/expanding it's use in a subsequent review)

Reply:
> Should this binary be set as a variable one can control? similar to PAGER or DELTA etc?

Good point, that does sound reasonable to me. I can see a use case here of users wanting to override the preview with something fancier and/or colorful.

> Are there other places in the code we care about looking at directories with a tool like this?

Currently no, but maybe in the future.

I've updated this PR accordingly. I think the change is small enough to have it included in this PR.